config/lint.xml

```diff
@@ -59,4 +59,10 @@
     <issue id="AlwaysShowAction" severity="ignore" />
 
     <issue id="InvalidFragmentVersionForActivityResult" severity="ignore" />
-</lint>
\ No newline at end of file
+
+    <!-- This incorrectly identifies places where String.format is used on the string: https://github.com/JakeWharton/timber/issues/434 -->
+    <issue id="TimberArgCount" severity="ignore" />
```

Comment:
This and the lint below both failed after the upgrade, although both seem to be false positives. I've filed a bug for one and referenced an existing one for the other.